pyproject.toml

```diff
@@ -368,6 +349,10 @@ exclude = [
 # Keep this one enabled
 "pandas/_typing.py" = ["TCH"]
 
+[tool.ruff.format]
+docstring-code-format = true
+docstring-code-line-length = 60
```

Comment:
Is that shorter than the normal line length on purpose?

Curious: where does the line start for doc-strings: at the actual line start or at the indentation level of the doc-string?

Reply:
> where does the line start for doc-strings:

This doesn't seem to be documented well. I think for a simple function line length starts at the actual line start, but for a class method docstring it starts somewhere else...that's why I chose a fixed number since the default lead to violating our 88 line length configuration